pandas/_libs/parsers.pyx

```diff
@@ -846,9 +844,9 @@ cdef class TextReader:
             passed_count = len(header[0])
 
             # if passed_count > field_count:
-            #     raise ParserError('Column names have %d fields, '
-            #                        'data has %d fields'
-            #                        % (passed_count, field_count))
+            #    raise ParserError('Column names have %d fields, '
+            #                      'data has %d fields'
+            #                      % (passed_count, field_count))
```

Comment:
can you get this back to zero-diff.  otherwise the next person who comes along to review will go through the same song-and-dance about it